### ISSUE TITLE
Windows Mapped Network Drive not found

### ISSUE THREAD

**Issue by quazi-moto** [SickChill/SickChill#5115]
So since my 'SickRage' install is completely gone and unable to recover, I installed the SickChill
I now need to to add shows again from my Z: drive, which is mapped window network drive to my TV shows folder on my NAS, but it is not showing in the list of possible locations. This used to work, not sure why I cannot select the mapped Z: drive location?

I checked, my Z: drive is available, online and navigable through explorer, just not showing in the directory location list in SickChill, help would be much appreciated.

Thanks guys, keep up the good work!

**Comment by quazi-moto** [comment]
So, somewhat good news, I was able to restore my database from a previous backup, win!

But even though all my shows and info are once again available (albeit a few months old), it is unable to update any show information since it still cannot find my Z: mapped network drive to my NAS.

**Comment by miigotu** [comment]
This is a permissions issue

**Comment by quazi-moto** [comment]
Thanks for your reply, but I'm not sure how that's the case?
On further investigation, in SickChill General Settings,  I was able to add my network drive folder's full location: "\\JOKER\Videos\TV Shows"
I was also able to update a TV shows directory e.g. "Z:\Castle" to "\\JOKER\Videos\TV Shows\Castle", this then works. However my mapped Z: to this very same location is still inaccessible. Again, this used to work, and it worked straight away when SickRage was first installed. 
To confirm, I just set my Z: security settings to "Everyone" to have "Full Access". Still no dice.
Should I somehow have SickChill launch as administrator perhaps? If so, how?
Thanks!
